src/components/rating/Rating.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+
+function Rating(props) {
```

Comment:
Damn bud!!! It looks great but you just had to copy the star from the lab text like this : ☆☆☆☆☆ xD and again, for this kinda questions take in mind that the switch statement is the better option

Overall, good job man!

```
function Rating(props) {
  
 let num = Math.round(props.children);

  let star = "";
  switch (num) {
    case 0:
      star = "☆☆☆☆☆";
      break;
    case 1:
      star = "★☆☆☆☆";
      break;
    case 2:
      star = "★★☆☆☆";
      break;
    case 3:
      star = "★★★☆☆";
      break;
    case 4:
      star = "★★★★☆";
      break;
    case 5:
      star = "★★★★★";
      break;
    default:
      break;
  }

  return <div>{star}</div>;
}
```